user_data/strategies/sample_strategy.py

```diff
@@ -103,8 +102,10 @@ def populate_indicators(self, dataframe: DataFrame, metadata: dict) -> DataFrame
         # ------------------------------------
 
         # ADX
-        dataframe['adx'] = ta.ADX(dataframe)
+        # dataframe['adx'] = ta.ADX(dataframe)
```

Comment:
It is ... ?

Reply:
Almost :)

Pls see my variant, sorry that I changed your branch -- that was more effective than explain it in words...

Reply:
that's fine :+1: 

ADX is now commented twice (that's what i didn't get) - but that's fine :)

Reply:
oops, and you did not fix it... (double comment)